frontend/src/views/coursemapper/ActiveCourse.js

```diff
@@ -125,7 +125,7 @@ const ActiveCourse = ({
       </div>
 
       <List className={classes.list}>
-        {orderedConcepts.map((concept, index) =>
+        {orderedConcepts.filter(concept => concept.level === 'OBJECTIVE').map((concept, index) =>
```

Comment:
This is the wrong place to do this. It needs to be done where `orderedConcepts` is created (the `useMemo()` call). It also needs to use `course.objectiveOrder` instead of `course.conceptOrder` as the order list parameter for `sortedConcepts()`